### ISSUE TITLE
Make system panel read only

### ISSUE THREAD

**Issue by Kelsey-Ethyca** [ethyca/fides#3889]

**Comment by rsilvery** [comment]
@TheAndrewJackson is this done?

**Comment by TheAndrewJackson** [comment]
@rsilvery Yes, I have a branch with this work finished. This was waiting on the schema changes to get merged into main. I can open a PR for this now.

**Comment by Roger-Ethyca** [comment]
moving to done 


![Image](https://github.com/ethyca/fides/assets/115798183/195d7a85-4af6-4859-a4f2-7bd6f6a70c48)
